src/scss/components/_logosSection.scss

```diff
@@ -47,7 +47,7 @@
   padding-top: 16rem;
   padding-bottom: 45.8rem;
 
-  @include breakpoint(md) {
+  @include breakpoint(lg) {
```

Comment:
there is an issue between 1024px to 1439px,
you can fix it by changing the breakpoint "lg" to "md"